common/changes/@kadena/client/refactor-tools-use-contract-parser_2023-08-02-08-36.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@kadena/client",
```

Comment:
I didn't change anything in the client! 
I don't know why rush added the change log for client instead of pactjs-generator!

Reply:
Maybe rebase on main again and then check whether `rush change` still wants you to add the comment